src/Command/ComposerJsonCommand.php

```diff
@@ -416,6 +416,14 @@ static function ($a, $b) {
                     return 1;
                 }
 
+                if ('composer-runtime-api' === $a) {
+                    return -1;
+                }
+
+                if ('composer-runtime-api' === $b) {
+                    return 1;
+                }
```

Comment:
```suggestion
                if (strpos($a, '/') === false && strpos($b, '/') !== false) {
                    return -1;
                }

                if (strpos($b, '/') === false && strpos($a, '/') !== false) {
                    return 1;
                }
```

How about sorting all dependencies that don’t have a slash in them after the PHP extensions?

Reply:
Good idea 👍